tests/README.md

```diff
@@ -0,0 +1,36 @@
+# Tests Directory
+
+This directory is where all product test suites, test cases, and test fixtures are defined.
+
+## Organization
+Tests will be organized into one of three top level categories: Functional, Cross Functional, or Libtest.
+
+### Functional
+Tests that are focused on one particular component or functionality of ODF will belong in the Functional module.
+
+An example of this kind of testing is our [mcg](https://github.com/red-hat-storage/ocs-ci/tree/master/tests/manage/mcg) testing. This module is aimed at testing one ODF component (MCG) and all of the tests in this module are aimed at testing one particular feature of that component (bucket_creation, multi_region, object_versioning, etc.).
+
+### Cross Functional
+Tests that target multiple components or perform actions that span different areas of the product will belong in the Cross Functional module.
+
+An example of this kind of testing is our [longevity](https://github.com/red-hat-storage/ocs-ci/tree/master/tests/e2e/longevity) testing. These test cases are not targeting one specific feature of the product. Rather, they are creating scenarios that represent real life user behavior which spans across several different product features/components. Each test case under this module performs very different actions from the others with regards to which components are involved. See [test_stage1.py](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/longevity/test_stage1.py) for a specific example.
```

Comment:
would the link path of https://github.com/red-hat-storage/ocs-ci/tree/master/tests/e2e/longevity change once we do the restructure?